sklearnex/decomposition/pca.py

```diff
@@ -1,5 +1,5 @@
 # ===============================================================================
-# Copyright 2021 Intel Corporation
+# Copyright 2023 Intel Corporation

```

Comment:
AFAIK it should be kept intact

Reply:
It was first created in 2023.

Reply:
Check commit history of this file, it wasn't created in 2023